ddtrace/debugging/_debugger.py

```diff
@@ -245,6 +247,12 @@ def __init__(self, tracer=None):
             raise_on_exceed=False,
         )
 
+        # TODO: this is only temporary and will be reverted once the DD_REMOTE_CONFIGURATION_ENABLED variable
+        #  has been removed
+        if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", True)) is False:
+            os.environ["DD_REMOTE_CONFIGURATION_ENABLED"] = "true"
```

Comment:
combined with the `if` above `DD_REMOTE_CONFIGURATION_ENABLED` will always be True, can we set it as such outside of the `if`?